packages/server/src/automations/steps/custom.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  ContextEmitter,
+  CustomStepInputs,
+  CustomStepOutputs,
+} from "@budibase/types"
+
+export async function run({
+  inputs,
+}: {
+  inputs: CustomStepInputs
+  appId: string
```

Comment:
Do we need these fields, if not used?

Reply:
Whoops, this was meant to be deleted. Remnants of a bad idea.